docs/fundamentals/code-analysis/configuration-files.md

```diff
@@ -97,7 +97,7 @@ Consider the following naming recommendations:
 - MSBuild tooling-generated global config files should be named *<%Target_Name%>_Generated.globalconfig* or similar.
 
 > [!NOTE]
-> The top-level entry `is_global = true` is required even when the file is named `.globalconfig`.
+> The top-level entry `is_global = true` is not required when the file is named `.globalconfig`, but it is recommended to include for clarity.
```

Comment:
```suggestion
> The top-level entry `is_global = true` is not required when the file is named `.globalconfig`, but it is recommended for clarity.
```

Reply:
Changed wording as suggested (amended).

I'm not familiar with the internals of dotnet releases. But in the dotnet PR I linked, the change was merged by @chsienki to a branch called `release/dev16.9` on 19 Dec 2020.